packages/plugin-rollup/src/__snapshots__/plugin.test.ts.snap

```diff
@@ -1,27 +1,28 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`snaps applies a transform 1`] = `
-"const foo = 'bar';
+"const foo = \\"bar\\";
```

Comment:
What's with the escaping here, that didn't happen with my PR? 🤔

Reply:
Babel changes `'` to `"` it seems?